packages/core/test/lib/create.js

```diff
@@ -17,136 +17,107 @@ describe("create", function () {
     config.artifactor = new Artifactor(config.contracts_build_directory);
   });
 
-  after("Cleanup tmp files", function (done) {
-    glob("tmp-*", (err, files) => {
-      if (err) done(err);
-      files.forEach(file => fse.removeSync(file));
-      done();
-    });
+  after("Cleanup tmp files", async () => {
```

Comment:
[Mocha says](https://mochajs.org/#arrow-functions) to use `function () {}` not `() => {}`:

```suggestion
  after("Cleanup tmp files", async function () {
```